### DISCUSSION TITLE
Use alternative to Python's `http.server`

### DISCUSSION THREAD

**Discussion by jens-br** [nest-desktop/nest-desktop#577] [General]
NEST Desktop's Python package uses Python's `http.server`. However, this is only recommended for testing purposes, as it has not many security options included (c.f. https://docs.python.org/3/library/http.server.html). Therefore, an alternative should be considered.

**Comment by babsey** [comment]
I misread the bin file. Actually, it doesn't start with `http.server`. 
In my view, the codes are fine. See the lines:

https://github.com/nest-desktop/nest-desktop/blob/3d1d5f7e70cda53fb070dd201fe3665d5c5e9fbd/bin/nest-desktop#L36

and

https://github.com/nest-desktop/nest-desktop/blob/3d1d5f7e70cda53fb070dd201fe3665d5c5e9fbd/nest_desktop/app.py#L12-L27
